root_skel/etc/NetworkManager/dispatcher.d/pre-up.d/00_filter.sh

```diff
@@ -7,4 +7,5 @@ subnet="$(ip -o a show dev "$device" scope global | awk '{ print $4 }')"
 iptables -F
 
 iptables -A OUTPUT -d "$subnet" -j ACCEPT
+iptables -A OUTPUT -d 127/8 -j ACCEPT
```

Comment:
```suggestion
iptables -A OUTPUT -d 127.0.0.0/8 -j ACCEPT
```

> I ran (on my machine) sudo iptables -A OUTPUT -d 127/8 -j ACCEPT which worked and sudo iptables -A OUTPUT -d 127./8 -j ACCEPT which failed. Therefore, this format should be valid.

Let's use the long format, just to be safe.